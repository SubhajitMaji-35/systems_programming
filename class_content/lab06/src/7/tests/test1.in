5 12 -3 7
0 8 14 -1
3 -6 9 4
2  4  6  8
1  3  5  7
9  0 -1 -2
