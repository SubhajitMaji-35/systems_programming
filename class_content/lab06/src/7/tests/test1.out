7 16 3 15 
1 11 19 6 
12 -6 8 2 
